lib/flipper/api/action.rb

```diff
@@ -88,13 +89,30 @@ def halt(response)
         throw :halt, response
       end
 
+      # Public: Call this with a json serializable object (i.e. Hash)
+      # to serialize object and respond to request
+      #
+      # object - json serializable object
+      # status - http status code
+
       def json_response(object, status = 200)
         header 'Content-Type', Api::CONTENT_TYPE
         status(status)
         body = JSON.dump(object)
         halt [@code, @headers, [body]]
       end
 
+      # Public: Call this with an ErrorResponse::ERRORS key to respond
+      # with the serialized error object as response body
+      #
+      # error_key - key to lookup error object
+      # status - http status code
+
+      def json_error_response(error_key, status)
```

Comment:
You're right, let's keep it simple for now, not really sure of a situation where it wouldn't be consistent anyways